lib/template/swift_source.rb

```diff
@@ -3,6 +3,8 @@ class Template::SwiftSource < Template::Base
 
   # cspell: disable
   CONTENT = <<-CONTENT
+    |import Foundation
```

Comment:
This file uses `URL` which comes from `Foundation`. Without this, we can't build/test this as a standalone package.